great_expectations/validator/metrics_calculator.py

```diff
@@ -177,6 +178,32 @@ def compute_metrics(
             runtime_configuration=runtime_configuration,
             min_graph_edges_pbar_enable=min_graph_edges_pbar_enable,
         )
+        return resolved_metrics, aborted_metrics_info
+
+    def compute_metrics(
+        self,
+        metric_configurations: List[MetricConfiguration],
+        runtime_configuration: Optional[dict] = None,
+        min_graph_edges_pbar_enable: int = 0,
+        # Set to low number (e.g., 3) to suppress progress bar for small graphs.
+    ) -> _MetricsDict:
+        """
+        Args:
+            metric_configurations: List of desired MetricConfiguration objects to be resolved.
+            runtime_configuration: Additional run-time settings (see "Validator.DEFAULT_RUNTIME_CONFIGURATION").
+            min_graph_edges_pbar_enable: Minumum number of graph edges to warrant showing progress bars.
+
+        Returns:
+            Dictionary with requested metrics resolved, with unique metric ID as key and computed metric as value.
+        """
+
+        # Note: Dropping aborted metrics for backward compatibility.
+        # This is a temporary solution until we can change all the callers to handle aborted metrics.
+        resolved_metrics, _ = self.compute_metrics_with_aborted_metrics(
```

Comment:
Refactoring to use the new method `compute_metrics_with_aborted_metrics` but throwing away the aborted info. That is the previous behavior, which is made more clear :) thanks